poisson_lib.py

```
@@ -0,0 +1,731 @@
+                "deviance": result.deviance,
+                "pearson_chi2": result.pearson_chi2,
+                "aic": result.aic,
+                "bic": result.bic
+            }
+        except (AttributeError, ZeroDivisionError) as e:
+            logger.debug(f"Failed to calculate Poisson fit stats: {e}")
+        
+        return result.params, result.conf_int(), result.pvalues, "OK", stats_metrics
+    
+    except Exception as e:
+        logger.exception(f"Poisson regression failed: {e}")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)